Readme.md

```diff
@@ -1,13 +1,40 @@
-# Please pardon our progress
-We are currently in the process of consolidating our repos.
-This Readme will be updated as we make significant changes.
+# Application Insights for .NET
+This is the .NET SDK for [Azure Monitor](https://docs.microsoft.com/azure/azure-monitor/overview) & [Application Insights](https://docs.microsoft.com/azure/azure-monitor/app/app-insights-overview).
+
+
+
+## Getting Started
+Please review our How-to guides to review which packages are appropriate for your project:
+- [.NET Console](https://docs.microsoft.com/azure/azure-monitor/app/console)
+- [ASP.NET](https://docs.microsoft.com/azure/azure-monitor/app/asp-net)
+- [ASP.NET Core](https://docs.microsoft.com/azure/azure-monitor/app/asp-net-core)
```

Comment:
Curious: Is there a .NET Core Console?

Reply:
Visual Studio does have a template for .NET Core Console Apps, but there's nothing special about them. they look just like other console projects.
We don't have a public doc for these project types, but I'm not certain we need one. :)

Reply:
Maybe we just replace `.NET Console` with `Console`? The document link also talked about `app/console` without specifying whether it is .NET Framework or .NET Core. In case the document is missing .NET Core part, it should be a bug on Azure Docs.

Reply:
Ah, I see what you're saying. Yes that makes more sense.